src/frontend/src/flows/displayUserNumber.ts

```diff
@@ -1,12 +1,21 @@
 import { html, render } from "lit-html";
+import { warningIcon } from "../components/icons";
 
 const pageContent = (userNumber: bigint) => html`
   <div class="container">
     <h1>Congratulations!</h1>
-    <p>
-      Please record your new Identity Anchor. You will need it later to use
-      Internet Identity or to add additional devices.
-    </p>
+    <p>Your new Identity Anchor has been created.</p>
+    <div class="nagBox">
+      <div class="nagIcon">${warningIcon}</div>
+      <div class="nagContent">
+        <div class="nagTitle">Record Your Identity Anchor</div>
+        <div class="nagMessage">
+          Do not forget to save this Identity Anchor. Save a backup on a storage
+          medium and write it down. You will need it later to use Internet
+          Identity or to add additional devices.
```

Comment:
Could we even strengthen the text a bit more and say e.g. the following? 
```suggestion
          Please record your new Identity Anchor. Keep a backup on a storage
          medium and write it down. You will need it later to use Internet
          Identity or to add additional devices. If you lose your Identity Anchor, 
          you will no longer be able to use it to authenticate to dApps and you 
          will lose access to them. 
```

Reply:
Hm, I'm unsure about the "...you will lose access to them.". You can still access dApps, just not with that identity.

Maybe like this?

> Please record your new Identity Anchor. Keep a backup on a storage medium and write it down. You will need it later to use Internet Identity or to add additional devices. If you lose your Identity Anchor, you will no longer be able to use this identity to authenticate to dApps and you will lose access to the associated profiles and data.

Reply:
Sounds great, thanks!